test/test_distributions.py

```diff
@@ -366,6 +371,48 @@ def get_sp_dist(jax_dist):
         np.broadcast_to(np.identity(3), (2, 3, 3)),
         None,
     ),
+    T(
+        dist.CAR,
+        1.2,
+        np.array([-0.2, 0.3]),
+        0.1,
+        np.array(
+            [
+                [0.0, 1.0, 1.0, 0.0],
+                [1.0, 0.0, 0.0, 1.0],
+                [1.0, 0.0, 0.0, 1.0],
+                [0.0, 1.0, 1.0, 0.0],
+            ]
+        ),
+    ),
+    T(
+        dist.CAR,
+        np.array([0.0, 1.0, 3.0, 4.0]),
+        0.1,
+        np.array([0.3, 0.7]),
+        np.array(
+            [
+                [0.0, 1.0, 1.0, 0.0],
+                [1.0, 0.0, 0.0, 1.0],
+                [1.0, 0.0, 0.0, 1.0],
+                [0.0, 1.0, 1.0, 0.0],
+            ]
+        ),
+    ),
+    T(
+        _SparseCAR,
+        np.array([[0.0, 1.0, 3.0, 4.0], [2.0, -1.0, -3.0, 2.0]]),
+        0.0,
+        0.1,
+        np.array(
+            [
+                [0.0, 1.0, 1.0, 0.0],
+                [1.0, 0.0, 0.0, 1.0],
+                [1.0, 0.0, 0.0, 1.0],
+                [0.0, 1.0, 1.0, 0.0],
+            ]
+        ),
+    ),
```

Comment:
We need skip some tests here:
+ variance is not implemented
+ we can't jit/grad `W` parameter of SparseCAR

```diff
 _DIST_MAP = {
@@ -1024,6 +1026,7 @@ def test_jit_log_likelihood(jax_dist, sp_dist, params):
         "_ImproperWrapper",
         "LKJ",
         "LKJCholesky",
+        "_SparseCAR",
     ):
         pytest.xfail(reason="non-jittable params")

@@ -1378,6 +1381,9 @@ def test_log_prob_gradient(jax_dist, sp_dist, params):

     eps = 1e-3
     for i in range(len(params)):
+        if jax_dist is _SparseCAR and i == 3:
+            # skip taking grad w.r.t. W
+            continue
         if isinstance(
             params[i], dist.Distribution
         ):  # skip taking grad w.r.t. base_dist
@@ -1508,6 +1514,8 @@ def test_mean_var(jax_dist, sp_dist, params):
     else:
         if jnp.all(jnp.isfinite(d_jax.mean)):
             assert_allclose(jnp.mean(samples, 0), d_jax.mean, rtol=0.05, atol=1e-2)
+        if isinstance(d_jax, dist.CAR):
+            pytest.skip("CAR distribution does not have `variance` implemented.")
         if jnp.all(jnp.isfinite(d_jax.variance)):
```